### DISCUSSION TITLE
filter by date

### DISCUSSION THREAD

**Discussion by mohammadjavad948** [SeaQL/sea-orm#984] [Q&A] [answered]
hello how can i cast dates for filter in postgres?

**Comment by billy1624** [comment, marked as answer]
Hey @mohammadjavad948, sorry for the delay. I don't think you need to perform casting. All you need to do is to construct a datetime value with either `chrono` or `time` crate. Then, apply filter on it.

```rs
use time::macros::{date, time};

let model: Vec<entity::Model> = Entity::find()
    .filter(entity::Column::DateTime.gt(date!(2022-03-13).with_time(time!(16:24:00))))
    .all(db)
    .await?;
```
